slsDetectorServers/jungfrauDetectorServer/slsDetectorFunctionList.c

```diff
@@ -468,6 +477,119 @@ int setDefaultDacs() {
     return ret;
 }
 
+int readConfigFile() {
+
+    if (initError == FAIL) {
+        return initError;
+    }
+
+    char fname[128];
```

Comment:
Do you want to use a constant for the size as below, to make sure there can be no mismatch?

Reply:
will do